libobs/obs-internal.h

```diff
@@ -756,7 +756,6 @@ struct obs_source {
 	volatile bool timing_set;
 	volatile uint64_t timing_adjust;
 	uint64_t resample_offset;
-	uint64_t last_audio_ts;
```

Comment:
Can we even safely remove this from the struct definition?

Reply:
Yep